packages/kolibri-sandbox/src/xAPI/xAPIConstants.js

```diff
@@ -2,8 +2,6 @@
  * xAPI Constants
  */
 
-import 'core-js/features/set';
```

Comment:
We were individually polyfilling specific features in the kolibri-sandbox code. Much better to use babel present env to handle this for us instead.